core/services/synchronization/stats_pusher.go

```diff
@@ -26,8 +26,18 @@ var (
 	})
 )
 
-// StatsPusher polls for events and pushes them via a WebSocketClient
-type StatsPusher struct {
+//go:generate mockery -name StatsPusher -output ../../internal/mocks/ -case=underscore
+
+// statsPusher polls for events and pushes them via a WebSocketClient
```

Comment:
Should we give some examples of what types of events are pushed? And what types of clients would consume these events?

Reply:
Also seems to be a typo, should be uppercase `StatsPusher`